src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx

```diff
@@ -43,6 +43,8 @@ export const DashboardTopBanner = (props: DashboardTopBannerProps) => {
       ),
       cta: {
         content: l10n.getString("dashboard-top-banner-protect-your-data-cta"),
+        // Ignored for test coverage; to be replaced by a link:
+        /* c8 ignore next 4 */
```

Comment:
It's a shame we're no longer using Istanbul for measuring coverage - that [had](https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md) `ignore if` and `ignore else` comments, and `ignore next` appeared to skip a whole node in the tree (e.g. a function block) rather than a single line.

But yes, it can get annoying with Prettier and having to go back because of CI rejecting your code. My rule of thumb was kinda "big code blocks use `start ... stop`, a couple of lines use `next`. But there's something to be said for not using `n` and always using `start ... stop` if it's more than a single line.

...in fact, I was on my way replacing them, but then I realised that that would increase the risk of future edits to the code staying within the `ignore` block, and then not get covered with unit tests. The advantage of using specific line numbers, is that future edits to the code will very likely trigger a reconsidering of whether that code shouldn't in fact need unit tests as well. 

So maybe a good rule of thumb is: `next n` if future edits are likely to need unit tests, and `start ... stop` if they don't?